OmniBLE/PumpManager/OmnipodPumpManager.swift

```diff
@@ -426,7 +428,7 @@ extension OmnipodPumpManager {
     }
     
     // MARK: Testing
```

Comment:
Disable this sim code that won't compile (due to changes in OmniBLE since it was originally written. I need the sim to run for the tests. This code doesn't matter for my testing through

Reply:
We should probably either fix this code or remove it. I haven't attempted to run it at all yet but at some point we'll probably want something that works for testing.

Reply:
Done - I removed this code, pulled in master, and the tests still passed.

Let me know if anything else needed on this PR to merge it or if further questions/concerns on the SPM things.